src/deriving.ml

```diff
@@ -59,6 +59,17 @@ let () =
 
 let keep_w60_impl () = !keep_w60_impl || Driver.pretty ()
 let keep_w60_intf () = !keep_w60_intf || Driver.pretty ()
+let allow_unused_code_warnings = ref Options.default_allow_unused_code_warnings
+
+let () =
+  Driver.add_arg "-deriving-allow-unused-code-warnings"
```

Comment:
I'm wondering if we can come up with a shorter name for this flag. Would `-deriving-allow-w32-w60` be clear?

Reply:
I think unused type warnings might also be included. I don't really want to limit the flag to certain warning numbers because we might want to extend it if more get added. Also the warning numbers are not super readable. But something like `-unused-code-warnings=true` and `-unused-code-warnings=false` would be fine if that looks better.

Reply:
Yes, that sounds good to me. It also makes a better cli-design IMO: With `-deriving-allow-unused-code-warnings` and `-deriving-disallow-unused-code-warnings`, one could pass in both those flags, and it would be unclear which one should dominate.